.env.example

```
@@ -26,3 +25,8 @@ CRYPTO__PROVIDER=file
 CRYPTO__ECDH_PRIVATE_KEY_PATH=tests/assets/alice.pfx
 # Additional certificates
 EXTRA_CA_CERTS=tests/assets/extra-ca-certs.pem
+# Pytest
+PYTEST__USE_TESTCONTAINERS=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PYTEST__USE_TESTCONTAINERS key should go before the S3__ACCESS_KEY key

(UnorderedKey)